src/west/commands/project.py

```diff
@@ -16,14 +16,65 @@
 from west import util
 from west.commands import WestCommand
 from west.manifest import default_path, Remote, Project, SpecialProject, \
-                          Manifest, MalformedManifest, META_NAMES
-
+                          MalformedManifest, META_NAMES
+from west.manifest import Manifest as WestManifest
 
 # Branch that points to the revision specified in the manifest (which might be
 # an SHA). Local branches created with 'west branch' are set to track this
 # branch.
 _MANIFEST_REV_BRANCH = 'manifest-rev'
 
+class Manifest(WestCommand):
+    def __init__(self):
+        super().__init__(
+            'manifest',
+            _wrap('''
+            Display or save the manifest file.
+
+            By default, displays the current manifest on stdout.
+            With additional options it can generate manifest variants based
+            on the state of the different projects and the options
+            provided.'''))
+
+    def do_add_parser(self, parser_adder):
+        return _add_parser(
+            parser_adder, self,
```

Comment:
I think a `-o` or `--output-file` argument would be nice here to avoid forcing the user to use shell redirection etc. to save to a file, but I don't think that should block the merge.